src/test/java/com/revature/studyforce/notification/service/StacktraceSubscriptionServiceTest.java

```diff
@@ -0,0 +1,143 @@
+package com.revature.studyforce.notification.service;
+
+import com.revature.studyforce.flashcard.model.Flashcard;
+import com.revature.studyforce.flashcard.repository.FlashcardRepository;
+import com.revature.studyforce.notification.model.FlashcardSubscription;
+import com.revature.studyforce.notification.model.StacktraceSubscription;
+import com.revature.studyforce.notification.model.Subscription;
+import com.revature.studyforce.notification.repository.FlashcardSubscriptionRepository;
+import com.revature.studyforce.notification.repository.StacktraceSubscriptionRepository;
+import com.revature.studyforce.stacktrace.model.Stacktrace;
+import com.revature.studyforce.stacktrace.repository.StacktraceRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@SpringBootTest
+@TestPropertySource(locations = "classpath:test-application.properties")
+public class StacktraceSubscriptionServiceTest {
```

Comment:
@Author Documentation required and @link to service layer being tested.

Reply:
fixed